packages/features/ee/workflows/components/WorkflowListPage.tsx

```diff
@@ -143,16 +144,41 @@ export default function WorkflowListPage({ workflows, profileOptions, hasNoWorkf
                             )}
                           </Badge>
                         </li>
-                        {workflow.teamId && (
-                          <li>
-                            <Badge variant="gray">
-                              <>{workflow.team?.name}</>
-                            </Badge>
-                          </li>
-                        )}
+                        <div className="block md:hidden">
+                          {workflow.team?.name && (
+                            <li>
+                              <Badge variant="gray">
+                                <>{workflow.team.name}</>
+                              </Badge>
+                            </li>
+                          )}
+                        </div>
                       </ul>
                     </div>
                   </Link>
+                  <div>
+                    <div className="hidden md:block">
+                      {workflow.team?.name && (
+                        <Badge className="mr-4 mt-1 p-[1px] px-2" variant="gray">
+                          <Avatar
+                            alt={workflow.team?.name || ""}
+                            href={
+                              workflow.team?.id
+                                ? `/settings/teams/${workflow.team?.id}/profile`
+                                : "/settings/my-account/profile"
+                            }
+                            imageSrc={getPlaceholderAvatar(
```

Comment:
@leog Why don't we access the team logo directly like that? In most other places the team avatars are actually not working with orgs, they only work in the settings shell where we also do it like that